queries.sql

```diff
@@ -139,3 +139,85 @@ JOIN animals a ON o.id = a.owner_id
 GROUP BY o.id, o.full_name
 ORDER BY COUNT(a.id) DESC
 LIMIT 1;
+
+
+-- Write queries to answer the following:
+
+    --  Who was the last animal seen by William Tatcher?
+    SELECT a.name AS animal_name
+    FROM visits v
+    JOIN animals a ON v.animal_id = a.id
+    JOIN vets vt ON v.vet_id = vt.id
+    WHERE vt.name = 'William Tatcher'
+    ORDER BY v.visit_date DESC
+    LIMIT 1;
+
+  --  How many different animals did Stephanie Mendez see?
+    SELECT COUNT(DISTINCT a.id)
+    FROM visits v
+    JOIN animals a ON v.animal_id = a.id
+    JOIN vets vt ON v.vet_id = vt.id
+    WHERE vt.name = 'Stephanie Mendez';
+
+    --  List all vets and their specialties, including vets with no specialties.
+    SELECT vt.name, COALESCE(array_agg(s.name), ARRAY[]::text[]) AS specialties
+    FROM vets vt
+    LEFT JOIN vet_specialties vs ON vt.id = vs.vet_id
+    LEFT JOIN specialties s ON vs.specialty_id = s.id
+    GROUP BY vt.id, vt.name;
+
+     --  List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+    SELECT a.name AS animal_name
+    FROM visits v
+    JOIN animals a ON v.animal_id = a.id
+    JOIN vets vt ON v.vet_id = vt.id
+    WHERE vt.name = 'Stephanie Mendez'
+    AND v.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
+
+    --  What animal has the most visits to vets?
+    SELECT a.name AS animal_name, COUNT(*) AS visit_count
+    FROM visits v
+    JOIN animals a ON v.animal_id = a.id
+    GROUP BY a.name
+    ORDER BY visit_count DESC
+    LIMIT 1;
+
+    --  Who was Maisy Smith's first visit?
+    SELECT a.name AS animal_name, MIN(v.visit_date) AS first_visit_date
+    FROM visits v
+    JOIN animals a ON v.animal_id = a.id
+    JOIN vets vt ON v.vet_id = vt.id
+    WHERE vt.name = 'Maisy Smith'
+    GROUP BY a.name;
```

Comment:
Please add the correct `query` and the `screenshot` 🤗🛠

![image](https://github.com/DevaWinner/vet-clinic/assets/78251927/7c240a88-7ea9-4742-ab43-7394c81fd1b8)